skops/io/tests/test_persist.py

```diff
@@ -67,6 +67,47 @@
 # Python 3.8)
 ATOL = 1e-6 if sys.platform == "darwin" else 1e-7
 
+# TODO: remove when these are added to trusted types
+SKLEARN_EXCLUDED_TYPES = (
```

Comment:
These are some of the `sklearn` types that are still untrusted. Therefore, I would like to exclude them when we look for `untrusted_types` in `test_can_persist_fitted`.

I don't think this is the _cleanest_ way to write something like this. Can you think of a better alternative?

Reply:
I think instead of doing this, we should check if any of the types reported by `get_untrusted_set` belong to `SKLEARN_ESTIMATOR_TYPE_NAMES`, and avoid having this list here.